tools/print_psd_content.cpp

```diff
@@ -31,6 +31,10 @@ class Delegate : public psd::DecoderDelegate {
       psd::color_mode_string(fileHeader.colorMode));
   }
 
+  void onLayerSelected(const std::string &layerName){
```

Comment:
Follow the existing style:
```suggestion
  void onLayerSelected(const std::string& layerName) {
```